desktop/main.js

```diff
@@ -246,7 +262,15 @@ const mainWindow = (() => {
     * */
     return app.whenReady()
         .then(() => {
-            const browserWindow = new BrowserWindow({
+            /*
+             * We only want to register the scheme this way when in dev, since
+             * when the app is bundled electron-builder will take care of it.
+            */
```

Comment:
Minor style, would be good to update lines 257-261 above in a similar way as well 
```suggestion
             /**
             * We only want to register the scheme this way when in dev, since
             * when the app is bundled electron-builder will take care of it.
             */
```

Reply:
Done